csharp/src/Google.Protobuf/Google.Protobuf.nuspec

```diff
@@ -24,7 +24,7 @@
       <group targetFramework="monotouch" />
       <group targetFramework="monoandroid" />
       <!-- Dependencies for newer, more granular platforms (.NET Core etc) -->
-      <group targetFramework="dotnet">
+      <group targetFramework="netstandard1.0">
         <dependency id="System.Collections" version="4.0.0" />
```

Comment:
These are the old/wrong dependency versions to use with netstandard. You have two choices:
1. Update to the 2.0-beta-* version of the refgen tool, which supports netstandard
2. Simply add a dependency to `netstandard.library 1.6.0` as that sets the baseline that you've compiled against. If you go this route, you don't need the refgen tool anymore.